http/headers/Permissions-Policy.json

```
@@ -659,6 +659,45 @@
             }
           }
         },
+        "gamepad": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Reference/Headers/Permissions-Policy/gamepad",
+            "spec_url": "https://www.w3.org/TR/gamepad/#permission-policy",
+            "support": {
+              "chrome": {
+                "version_added": "86",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "#restrict-gamepad-access",
+                    "value_to_set": "Enabled"
+                  }
+                ]
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": false
```

Maybe 17.4 according to https://webkit.org/blog/14916/release-notes-for-safari-technology-preview-186/ (and assuming it then went into what I believe is the corresponding release)

```suggestion
                "version_added": "17.4"
```